src/modules/data/views/Transactions/TransactionDetails.tsx

```diff
@@ -14,243 +14,153 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
```

Comment:
![image](https://user-images.githubusercontent.com/17093033/151999170-1b9da2fe-1c50-4085-972d-ed9580411f43.png)
Remove block is id is undefined